packages/allow-scripts/src/setup.js

```diff
@@ -2,73 +2,133 @@ const { existsSync,
         appendFileSync,
         readFileSync,
         writeFileSync,
-        createWriteStream,
       } = require('fs')
 const { spawnSync } = require('child_process')
 const path = require('path')
+const { FT } = require('./toggles')
+
+const NPM = {
```

Comment:
I note some overlap and differences with new implementation for #387 so I guess this might see some structural changes soon again?

(Still seems good to not overthink it, to consider this in isolation, and take that as it becomes necessary)